PeerioTechnologies/peerio-desktop
# fix: show file name for the first uploaded file

previewNextFile() was not being called after receiving non-empty files
prop.

#### Relevant info and issue/PR links 
 
#### Testing instructions  

Upload file(s) to chat. Check that file name of the first file is there in the upload dialog.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
